templates/corporate/billing.html

```diff
@@ -34,7 +34,14 @@ <h1>{{ _("Billing") }}</h1>
                         {% else %}
                         <p>Your current plan is <strong>{{ plan_name }}</strong>.</p>
                         {% endif %}
-                        <p>You are using <strong>{{ licenses_used }} of {{ licenses }} licenses</strong>.</p>
+                        <p>
+                            You are using <strong>{{ licenses_used }} of {{ licenses }} licenses</strong>.
+                            {% if automanage_licenses %}
+                            Your licenses are automatically managed by Zulip.
+                            {% else %}
+                            Your licenses are managed manually. Contact support to update licenses.
```

Comment:
Same here, we should be explicit: "You will not be able to add new users to your organization once you are using all of your licenses."

(And we should make sure the error messages from the product for that situation are clear)

Reply:
We do not block invites or user signup in general based on licenses number though, so should be change that to go with this?

Reply:
Let's put that change on @hackerkid's TODO list for manual license management.  For nowe we can not discuss that detail here.